scripts/check_format.sh

```diff
@@ -41,7 +41,7 @@ function check_shogun_style {
             fi
         done
 
-        RESULT_OUTPUT="$(git clang-format-3.8 --commit $BASE_COMMIT --diff --binary `which clang-format-3.8` $LIST)"
```

Comment:
need to get the Docker image fixed for this on travis

Reply:
Yeah, sorry, i didnt want to commit that change